src/ApiRequester.php

```diff
@@ -166,18 +165,9 @@ public function send()
         );
 
         // Defining Variables
-        $serverUrl = null;
-        $basePath = "";
-        $pathName = "";
-        if ($this->swaggerSchema->getSpecificationVersion() === '3') {
-            $serverUrl = $this->swaggerSchema->getServerUrl();
-        } else {
-            $httpSchema = $this->swaggerSchema->getHttpSchema();
-            $host = $this->swaggerSchema->getHost();
-            $basePath = $this->swaggerSchema->getBasePath();
-            $pathName = $this->path;
-            $serverUrl = "$httpSchema://$host$basePath$pathName$paramInQuery";
-        }
+        $serverUrl = $this->swaggerSchema->getServerUrl() . $paramInQuery;
```

Comment:
`$paramInQuery` should be removed here

Reply:
Good catch!! Thanks! However I realized that I dont have any test for parameters in query. I have first do this tests then remove this parameter. Probably this weekend.